{
  "schema_version": "1.4.0",
  "id": "GHSA-g358-g2pq-c46j",
  "modified": "2025-08-08T16:50:05Z",
  "published": "2025-08-08T12:32:17Z",
  "aliases": [
    "CVE-2025-53606"
  ],
  "summary": "Apache Seata: Deserialization of untrusted Data in Apache Seata Server",
  "details": "Deserialization of Untrusted Data vulnerability in Apache Seata (incubating).\n\nThis issue affects Apache Seata (incubating): 2.4.0.\n\nUsers are recommended to upgrade to version 2.5.0, which fixes the issue.",
  "severity": [
    {
      "type": "CVSS_V4",
      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:H/VA:H/SC:N/SI:N/SA:N/E:U"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "Maven",
        "name": "org.apache.seata:seata-serializer-fury"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "2.4.0"
            },
            {
              "fixed": "2.5.0"
            }
          ]
        }
      ],
      "versions": [
        "2.4.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2025-53606"
    },
    {
      "type": "WEB",
      "url": "https://github.com/apache/incubator-seata/commit/d2a18aef82c08535e4134642070c39d98654f0f6"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/apache/incubator-seata"
    },
    {
      "type": "WEB",
      "url": "https://lists.apache.org/thread/ggfd72vvvxjozs81zbcls45zxg64pphx"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-502"
    ],
    "severity": "HIGH",
    "github_reviewed": true,
    "github_reviewed_at": "2025-08-08T16:50:05Z",
    "nvd_published_at": "2025-08-08T10:15:26Z"
  }
}